includes/compat/class-wc-stripe-subscriptions-helper.php

```diff
@@ -14,6 +14,18 @@ class WC_Stripe_Subscriptions_Helper {
 	 */
 	private const DETACHED_SUBSCRIPTIONS_TRANSIENT_KEY = 'wcstripe_detached_subscriptions';
```

Comment:
Given that we're now using the database cache, should we rename the constant and remove the prefix?
```suggestion
	private const DETACHED_SUBSCRIPTIONS_CACHE_PREFIX = 'detached_subscriptions';
```